sort-selection/CHANGELOG.md

```diff
@@ -1,17 +1,21 @@
 # Changelog
-All notable changes to the "sort-selection" extension will be documented in this file.
+All notable changes to the "Sort delimited list" extension will be documented in this file.
+
+## 2.0.2 - 2021-04-29
+- Added extension icon;
+- Extension renamed from "Sort selection" to "Sort delimited list".
 
 ## 2.0.1 - 2019-05-12
-- Unified scripts
-- Supported unified natural sort
+- Unified scripts;
```

Comment:
I've never notices usage of semicolon as list separator in combination with bullet to denote list items before. It's an interesting detail - I found good article on the topic, and based on that, it looks to me like there should be no punctuation after these bullet list items. What do you think?

See: https://www.instructionalsolutions.com/blog/bulleted-list-punctuation

Reply:
It indeed does not happen in the texts in general. 

The semicolon is mostly used in Linux desktop files for actions, categories, keywords and mime-types to list the items. You can see the GNOME documentation about it; https://developer.gnome.org/integration-guide/stable/desktop-files.html.en

Take my Linux desktop files and check categories, keywords and mime-types:
[linux-desktop-files.zip](https://github.com/jmredfern/vscode-sort-selection/files/6448266/linux-desktop-files.zip)

We, Linux perfectionists, like to keep the desktop files organised with alphabetic sort.

Reply:
Very interesting! That makes sense in the files you showed since they're lists on a single line. In this case the list is not a single line but a bulleted list, so I think your example doesn't match? The link I pasted above does talk about specific cases where semicolons are used in bulleted lists, but I *think* this for case, no punctuation. Btw, I'm very happy to be proven wrong! :)

Reply:
I understood what you meaned, I must apply the grammar in the changelog and the README, right?